src/routes/rules/RulesView.vue

```diff
@@ -421,12 +448,19 @@ export default {
   margin-top: 64px;
 }
 
+.section#multiplayer {
+  margin-top: 121px;
```

Comment:
Prefer multiples of 8 for consistency and a regular grid size
```suggestion
  margin-top: 120px;
```